myriad-scheduler/src/test/java/org/apache/myriad/MyriadTestModule.java

```diff
@@ -78,14 +78,7 @@ protected void configure() {
   @Provides
   @Singleton
   ExecutorCommandLineGenerator providesCLIGenerator(MyriadConfiguration cfg) {
-    ExecutorCommandLineGenerator cliGenerator = null;
-    MyriadExecutorConfiguration myriadExecutorConfiguration = cfg.getMyriadExecutorConfiguration();
-    if (myriadExecutorConfiguration.getNodeManagerUri().isPresent()) {
```

Comment:
I am sure I am missing something, but I don't see how we're handling the lack of a NodeManagerUri in the refactored code. Please confirm how we are doing that.